.github/workflows/gradle.yml

```diff
@@ -397,8 +397,6 @@ jobs:
     steps:
       - name: Checkout Airbyte
         uses: actions/checkout@v3
-        with:
```

Comment:
:information_source: I've talked to some Chromatic people, who confirmed that this should only be needed if we do screenshot comparison, which we're not using in chromatic.